frontend/src/components/VIcon/VIcon.vue

```diff
@@ -52,6 +54,8 @@ const props = withDefaults(
   }
 )
 
+const { effectiveColorMode } = useDarkMode()
```

Comment:
I think there's potentially a performance concern with evoking this composable every time we use an icon. 

Instead, could this be outside of the component with either a prop for the effective color mode, or simply by changing the icon name? I think the only places it would need to be implemented are:

```
components/VLicense/VLicenseElements.vue
54:        :name="`licenses/${element}`"

components/VLicense/VLicense.vue
59:        :name="`licenses/${name}`"
```

Reply:
Fair point, I have moved the dark-mode composable one level up, into `VLicense` and `VLicenseElements` in 7842ef0.